docs/example-eatapple.md

```diff
@@ -22,7 +22,7 @@ variables such as `employee`, `company` and `tax`.
 
 The second component of a smart legal template is the model, which is
 expressed using the [Composer Concerto modeling
```

Comment:
The name for it should also be changed. Please remove`Composer` everywhere referring to the Concerto modelling language.